### PR TITLE
add bicep for module 01

### PR DESCRIPTION
**Pull request by jessica-ern**
Maybe this could be used as an answer key or like the "catchup scripts"

Addresses
https://github.com/percebus/azure-secure-networking-for-devs/issues/10

### CODE REVIEW

File: bicep/01/main.bicep [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  scope: hubRG
+  params: {
+    prefix: prefix
+    hubLocation: hubLocation
+    // addresses start at 10.1.0.0
+  }
+}
+
+resource spokeRGs 'Microsoft.Resources/resourceGroups@2023-07-01' = [for (spokeLocation, i) in spokeLocations: {
+  name: '${prefix}-spoke-${spokeLocation}-${i+1}-rg'
```
Comment (percebus) [inline bicep/01/main.bicep:28]: The idea of this hash is that 
- is unique to your stack
- And `hub` & `spoke` stacks could potentially share them

i.e. `v3` or `ERTYUI`

File: bicep/01/main.bicep [hunk 2]
```diff
@@ -0,0 +1,64 @@
+@description('The unique short name of the project')
+param prefix string
```
Comment (percebus) [inline bicep/01/main.bicep:2]: Based on this comment
https://github.com/percebus/azure-secure-networking-for-devs/pull/19/files#r1854060806

You could add something like `id` or `suffix`

Comment (jessica-ern) [reply bicep/01/main.bicep:2]: I was trying to match the naming conventions in the docs which doesn't use a suffix, only the prefix.

the ids auto-increment in the spoke loop

Comment (jessica-ern) [reply bicep/01/main.bicep:2]: (discussed outside of PR, misunderstanding on my side.  id is a shared unique identifier of the stack)

**Comment by jessica-ern** [comment]
Moving this PR of my fork and into the main repo

**Pull request closed without merging**